langchain/src/tools/index.ts

```diff
@@ -38,3 +38,4 @@ export {
 export { AIPluginTool } from "./aiplugin.js";
 export { ReadFileTool, WriteFileTool } from "./fs.js";
 export { BraveSearch, BraveSearchParams } from "./brave_search.js";
+export { StartExecutionAWSSfnTool, DescribeExecutionAWSSfnTool, SendTaskSuccessAWSSfnTool } from "./aws_sfn.js";
```

Comment:
This unfortunately can't be in the main entrypoint since it requires a third-party dep:

https://github.com/hwchase17/langchainjs/blob/main/.github/contributing/INTEGRATIONS.md#creating-a-separate-entrypoint